.devcontainer/devcontainer.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "Zipline CodeSpace",
+  "dockerComposeFile": "docker-compose.yml",
+  "service": "app",
+  "workspaceFolder": "/workspace",
+  "forwardPorts": [3000, 5432],
+  "features": {
+    "ghcr.io/devcontainers/features/common-utils:2": {},
+    "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
+    "ghcr.io/devcontainers/features/node:1": {}
+  },
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "[typescript]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode"
+        },
+        "[typescriptreact]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode"
+        },
+        "editor.defaultFormatter": "esbenp.prettier-vscode",
+        "editor.formatOnSave": true,
+        "files.autoSave": "afterDelay"
+      },
+      "extensions": ["prisma.prisma", "esbenp.prettier-vscode", "dbaeumer.vscode-eslint"]
+    }
+  },
+"remoteUser": "node",
+"remoteEnv": {
+  "CORE_DATABASE_URL": "postgres://postgres:postgres@localhost/zip10"
```

```suggestion
  "CORE_DATABASE_URL": "postgres://postgres:postgres@localhost/postgres",
  "DATABASE_URL": "postgres://postgres:postgres@localhost/postgres"
```

Add both if the user runs any Prisma commands in the shell.